sphinxserve/__init__.py

```diff
@@ -48,6 +52,13 @@
                         short: d
                         action: store_true
                         default: __SUPPRESS__
```

Comment:
If you're asking why I didn't remove `debug`: that's only for preserving backward compatibility.  If you're asking why I added `--loglevel`, it's because, like in all communication, it's useful to be able to limit how much is said depending upon what one's audience needs to know.

Before this PR, you have only the choice between seeing nothing and seeing everything; after this PR, you can see build results without the questionably-useful logging messages from gevent, but if you were to care about those because you were troubleshooting a problem, you could use an easy command-line argument to enable their display.